src/lib/component/partial/SigninDialog/SigninForm.js

```diff
@@ -20,7 +20,10 @@ const SignInForm = ({ disabled, error, onSubmit }) => {
 
   const handleSubmit = useCallback(
     ev => {
-      onSubmit({ username, password });
+      onSubmit({
+        username: username.trim(),
+        password: password.trim(),
```

Comment:
Does it make sense to trim whitespace from a password?

Reply:
whoops